### ISSUE TITLE
groupBy cannot accept bind params

### ISSUE THREAD

**Issue by ioncube** [phalcon/cphalcon#10748]
In 2.0.5, groupBy() cannot accept a bind params arg, even though what might be grouped by could contain an expression. Values therefore need to be bound using bind() or piggybacked onto the bind params of another method. Bind params should be added for consistency.

**Comment by ioncube** [comment]
Hi Andres. I saw that you tagged as NSTR. I assumed that this was a lack of feature and so a test case is irrelevant. Just so everyone is clear, is it the case that groupBy() is implemented such that it should already be accepting parameters to bind against?

**Comment by andresgutierrez** [comment]
Are you getting an error when you specify a bound parameter there?

**Comment by ioncube** [comment]
Maybe I made a mistake, but I had a clause such as:
->groupBy('floor(unix_timestamp(a) / :b:)', ['b'=>$b])

following the pattern of other methods I was using such as where(), andWhere() etc.
As $b didn't get bound to :b:, I had an error that the count of parameters was incorrect as fewer parameters were bound than necessary. One fix was binding 'b' in the bind list for one of the where conditions I had, but in the end I decided it was better to bind everything with a bind(), which I knew existed but wasn't documented in the builder section, hence the suggestion to add that.
